src/utils.c

```diff
@@ -488,3 +488,26 @@ static int android_close(void *cookie)
     return 0;
 }
 #endif  // PLATFORM_ANDROID
+
+// From https://stackoverflow.com/questions/1659440/32-bit-to-16-bit-floating-point-conversion/60047308#60047308
+
+unsigned int AsUnsignedInt(const float x) {
+    return *(unsigned int*)&x;
+}
+float AsFloat(const unsigned int x) {
+    return *(float*)&x;
+}
+
+float HalfToFloat(unsigned short x) {
+    const unsigned int e = (x&0x7C00)>>10; // exponent
+    const unsigned int m = (x&0x03FF)<<13; // mantissa
+    const unsigned int v = AsUnsignedInt((float)m)>>23; // evil log2 bit hack to count leading zeros in denormalized format
+    return AsFloat((x&0x8000)<<16 | (e!=0)*((e+112)<<23|m) | ((e==0)&(m!=0))*((v-37)<<23|((m<<(150-v))&0x007FE000))); // sign : normalized : denormalized
+}
+
+unsigned short FloatToHalf(float x) {
```

Comment:
Considering this function is only used on `rtextures` module, I'd prefer to move it there as a internal `static` function, it will also remove the additional coupling to `utils` module.